packages/mdctl-cli/tasks/study.js

```diff
@@ -233,15 +237,36 @@ class Study extends Task {
     
     Usage: 
       
-      mdctl study [command]       
+      mdctl study [command] --manifestObject     
           
     Arguments:               
       
       Command                        
         export - Exports the study from the current org
         import - Imports the study into the current org
         task [action] - Allows the select of tasks to export from the current org  
-        consent [action] - Allows the select of consent templates to export from the current org                                                                                                                                   
+        consent [action] - Allows the select of consent templates to export from the current org  
+        
+      Options 
+        
+        --manifestObject - receives a valid manifest JSON object or the path to a manifest file 
```

Comment:
I'd add that this is only valid during `export` command, and it can only contain object instances, other org config objects can be exported through regular `mdctl env export` command

Reply:
hi @nahueld , I added that in the "Notes" section but it makes sense to have it as part of the explanation of this flag. Thanks for that